ui/app/index.html

```diff
@@ -9,5 +9,14 @@
   <body>
     <div id="app"></div>
     <script type="module" src="/src/main.js"></script>
+    <script src='https://storage.ko-fi.com/cdn/scripts/overlay-widget.js'></script>
+    <script>
+      kofiWidgetOverlay.draw('jtf191', {
+        'type': 'floating-chat',
+        'floating-chat.donateButton.text': 'Donate',
+        'floating-chat.donateButton.background-color': '#00b9fe',
+        'floating-chat.donateButton.text-color': '#fff'
+      });
+    </script>
```

Comment:
This has got to go direct into the base html?

Reply:
not necessarily. Let me see if I can move the script

Reply:
@kruzaavn I moved it to the component. Looks a bit cleaner.
Still needs some styling, but if you're good with it we can deploy so people can start donating